Content/dotnet-new-nunit-fsharp/Company.TestProject1.fsproj

```diff
@@ -9,9 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="nunit" Version="3.9.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.9.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
+    <PackageReference Include="nunit" Version="3.10.0" />
```

Comment:
NUnit 3.10.0 had a critical bug - I'd recommend going straight to 3.10.1. 😄

Reply:
thanks! done)